### DISCUSSION TITLE
Getting version error while installing monkeytype locally

### DISCUSSION THREAD

**Discussion by dev-shetty** [monkeytypegame/monkeytype#4156] [Questions] [answered]
I am trying to run monkeytype locally on my machine but when I do `npm run install-all`,  and also getting a similar error while running `npm install`. **P.S. I only want to run the front end.**

I am using
```
OS: Garuda Linux
node-version: 18.14.2
npm-version: 9.5.0
```

![Image showing the error](https://user-images.githubusercontent.com/89470104/230730983-1ab94843-7488-4a5d-85ac-2f299ef8c842.png)

**Comment by Miodec** [comment, marked as answer]
You have to downgrade. Recommend using `nvm` for that.

**Comment by dev-shetty** [comment]
Thanks using `nvm` worked... 
This was the first time, me using (hearing) about `nvm`... it's a great tool.


**Also in docs it mentions firebase as optional but I couldn't run the frontend server without firebase... It is working now after I set firebase and followed the steps mentioned here -  [Setting up firebase docs](https://github.com/monkeytypegame/monkeytype/blob/master/CONTRIBUTING_ADVANCED.md#firebase-optional)**
